src/fontra/views/applicationsettings/panel-shortcuts.js

```diff
@@ -0,0 +1,462 @@
+import * as html from "../core/html-utils.js";
+import { addStyleSheet } from "../core/html-utils.js";
+import { BaseInfoPanel } from "./panel-base.js";
+import {
+  getActionIdentifiers,
+  getActionInfo,
+  getActionTitle,
+  getBaseKeyFromKeyEvent,
+  getShortCut,
+  getShortCutRepresentation,
+  getShortCuts,
+  setCustomShortCuts,
+  shortCutModifierMap,
+} from "/core/actions.js";
+import { translate } from "/core/localization.js";
+import { commandKeyProperty, isMac } from "/core/utils.js";
+import { IconButton } from "/web-components/icon-button.js"; // required for the icon buttons
+import { dialog, message } from "/web-components/modal-dialog.js";
+
+function getShortCutsGrouped() {
+  const shortCutsGrouped = {};
+  for (const actionIdentifier of getActionIdentifiers()) {
+    const actionInfo = getActionInfo(actionIdentifier);
+    const topic = actionInfo.topic || "9999-shortcuts.other";
+    if (!shortCutsGrouped[topic]) {
+      shortCutsGrouped[topic] = [];
+    }
+    shortCutsGrouped[topic].push(actionIdentifier);
+  }
+
+  // sort the actions by sortIndex
+  for (const topic in shortCutsGrouped) {
+    shortCutsGrouped[topic].sort((a, b) => {
+      const actionInfoA = getActionInfo(a);
+      const actionInfoB = getActionInfo(b);
+      return actionInfoA.sortIndex - actionInfoB.sortIndex;
+    });
+  }
+
+  return shortCutsGrouped;
+}
+
+addStyleSheet(`
+.fontra-ui-shortcuts-panel {
+  background-color: var(--ui-element-background-color);
+  border-radius: 0.5em;
+  padding: 1em;
+}
+.fontra-ui-shortcuts-panel-header {
+  font-weight: bold;
+}
+.fontra-ui-shortcuts-panel-buttons {
+  display: grid;
+  grid-template-columns: max-content max-content max-content;
+  gap: 1em;
+}
+`);
+
+export class ShortCutsPanel extends BaseInfoPanel {
+  static title = "shortcuts.title";
```

Comment:
I know we're not (yet) doing this in the fontinfo view, but let's use a prefix for all translatable string keys for this view.

```suggestion
  static title = "application-settings.shortcuts.title";
```

(This obviously also requires changes to the language files)